meterfs/meterfs.c

```diff
@@ -75,14 +74,6 @@ static uint32_t meterfs_calcChecksum(const void *buff, size_t bufflen)
 	return checksum;
 }
 
-
-static void meterfs_powerctrl(int state, meterfs_ctx_t *ctx)
-{
-	if (ctx->powerCtrl != NULL)
-		(ctx->powerCtrl)(state);
-}
```

Comment:
This wrapper should stay, it made `powerCtrl` method optional. Or is there a reason that it might be better to remove it?

Reply:
Nope, good catch, I Just went too aggressive on refactor